src/main/java/uk/gov/hmcts/reform/draftstore/endpoint/v3/DraftController.java

```diff
@@ -46,6 +54,11 @@ public DraftController(DraftStoreDAO draftRepo, UserIdentificationService userId
     }
 
     @GetMapping(path = "/{id}")
+    @ApiOperation("Find draft by ID")
+    @ApiResponses({
+        @ApiResponse(code = 200, message = "Success"),
+        @ApiResponse(code = 404, message = "Not found", response = ErrorResult.class),
+    })
     public Draft read(
```

Comment:
Doesn't this need a MediaType.ALL_VALUE <- something like that
because you've said everything consumes json at the class level but on a GET there's no consumes?

Reply:
fixed